modules/hooks/templates.js

```diff
@@ -4,34 +4,34 @@ export default function()
 {
     Hooks.on("updateMeasuredTemplate", (template, data, options, user) => 
     {
-        if (game.user.id == user && (data.x || data.y))
+        if (game.user.isUniqueGM && (data.x || data.y) && !data.flags?.wfrp4e?.preventRecursive)
```

Comment:
Where is the recursion coming from here? I don't even see `preventRecursive` in this PR

Reply:
removed, it was something old from my first appraoch to area effects.